### PR TITLE
Feature muboot

### PR DESCRIPTION
**Pull request by flbulgarelli**
See https://github.com/mumuki/mumukit-platform/wiki/%5BMuBoot%5D-Standard-Mumuki-Platform-Boot-Process

### CODE REVIEW

File: devstart [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+echo '/    |    \  |   |  |  |  \  \__\  ___/ '
+echo '\_______  /__|   |__|  |__|\___  >___  >'
+echo '        \/                     \/    \/ '
+echo ''
+
+echo "[MumukiDevstart::Office] Installing dependencies...."
+bundle install --quiet
+
+echo "[MumukiDevstart::Office] Starting server...."
+rails s -p 3005
```
Comment (faloi) [inline devstart]: Why is the port here AND in `puma.rb`?

File: config/puma.rb [hunk 2]
```diff
@@ -1,47 +1,15 @@
...
-#
-port        ENV.fetch("PORT") { 3000 }
+preload_app!
 
-# Specifies the `environment` that Puma will run in.
-#
-environment ENV.fetch("RAILS_ENV") { "development" }
+rackup      DefaultRackup
+port        ENV['MUMUKI_OFFICE_PORT'] || 3005
+environment ENV['RACK_ENV']           || 'development'
```
Comment (faloi) [inline config/puma.rb]: Is it ok to stop using `RAILS_ENV`?

File: config.ru [hunk 3]
```diff
@@ -1,5 +1,14 @@
-# This file is used by Rack-based servers to start the application.
+## Development defaults
+if ENV['RAILS_ENV'] == 'development' || ENV['RACK_ENV'] == 'development'
```
Comment (faloi) [inline config.ru]: Why both?

File: README.md [hunk 4]
```diff
@@ -17,9 +17,138 @@ Mumuki Office is a RESTful service and GUI that allows to
...
 
-## Basic Platform Permissions Setup
+## Preparing environment
+
+### TL;DR install
+
+1. Install [Vagrant](https://www.vagrantup.com/downloads.html) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+2. Run `curl https://raw.githubusercontent.com/mumuki/mumuki-devinstaller/master/install.sh | bash`
+3. `cd mumuki && vagrant ssh` and then - **inside Vagrant VM** - `cd /vagrant/office`
+4. Go to Install Section
```
Comment (faloi) [inline README.md]: It's not clear which is that section. Section 5?

File: README.md [hunk 5]
```diff
@@ -17,9 +17,138 @@ Mumuki Office is a RESTful service and GUI that allows to
...
+
+### Installing the server
+
+If you just want to install dependencies, just do:
+
+```
+bundle install
+```
+
+### Running the server
```
Comment (faloi) [inline README.md]: I've seen this before... Why not linking to a wiki entry on mumuki-platform?

File: Gemfile [hunk 6]
```diff
@@ -31,7 +31,7 @@ gem 'mumukit-nuntius', '~> 5.0'
 gem 'mumukit-auth', '~> 7.0'
 gem 'mumukit-core', '~> 1.0'
 gem 'mumukit-login', '~> 3.0'
-gem 'mumukit-platform', '~> 0.1'
+gem 'mumukit-platform', '~> 0.4'
```
Comment (faloi) [inline Gemfile]: I don't know what's this, but doesn't seem related to muboot.

File: devstart (hunk 1 above)
Comment (flbulgarelli) [inline devstart]: Because rails and rackup do not honor it properly.  See https://github.com/mumuki/mumukit-platform/wiki/%5BMuBoot%5D-Standard-Mumuki-Platform-Boot-Process: 

> Environment-variables configuration must have more priority than harcoded defaults, and server-boot command like rackup and puma must have more priority than environment-variables.
As an exception, rackup command will not honor port declaration using env-variables.

File: Gemfile (hunk 6 above)
Comment (flbulgarelli) [inline Gemfile]: It is, since that previous platform version did not handle the `MUMUKI_ORGANIZATION_MAPPING` properly

File: README.md (hunk 5 above)
Comment (flbulgarelli) [inline README.md]: Because I want a README that is very easy to read. Making you following links and reading generic explanations is error prone. 

Having a nearly copy-pasted installations instructions is error prone too. But this section should not change frequently, while it will be read lots of times by each developer.

File: README.md (hunk 4 above)
Comment (flbulgarelli) [inline README.md]: Its the section whose title is 'Installing and running'. I can change that.

File: config.ru (hunk 3 above)
Comment (flbulgarelli) [inline config.ru]: Because although RACK_ENV is the standard variable, RAILS_ENV is a pretty common non-standard variable. Reading both improves compatibility
